src/png.toit

```diff
@@ -51,7 +56,10 @@ class Png extends CustomElement:
       --id/string?=null
       --background=null
       --border/Border?=null
-      --png-file/ByteArray:
+      --png-file/ByteArray
+      --color/int?=null
+      --palette-transformer/Lambda?=null:
```

Comment:
Are you sure this wouldn't be better as some reasonable interface? The raw lambda just seems a bit hard to understand and hard to type. Also it seems rather inefficient to have to pass back the 4-tuple byte arrays.